Sources/MapboxCoreNavigation/RouteController.swift

```diff
@@ -247,7 +247,7 @@ open class RouteController: NSObject {
     public func locationManager(_ manager: CLLocationManager, didUpdateLocations locations: [CLLocation]) {
         guard let location = locations.last else { return }
         
-        guard delegate?.router(self, shouldDiscard: location) ?? DefaultBehavior.shouldDiscardLocation else {
+        guard !(delegate?.router(self, shouldDiscard: location) ?? DefaultBehavior.shouldDiscardLocation) else {
```

Comment:
Maybe we should also invert `DefaultBehavior.shouldDiscardLocation` to maintain existing 'default' behavior? This will not only also make the fix 'less breaking' since many users rely on defaults but also won't require that many unit tests changes below

Reply:
Great idea! I will give it a try.